dev/tests/integration/phpunit.xml.dist

```diff
@@ -19,7 +19,6 @@
         <testsuite name="Magento Integration Tests">
             <directory suffix="Test.php">testsuite</directory>
             <directory suffix="Test.php">../../../update/dev/tests/integration/testsuite</directory>
-            <exclude>testsuite/Magento/Test/Integrity</exclude>
```

Comment:
Why would you like to combine these tests with all integration tests?
I'd assume that integrity tests give more general picture, so their results would make sense standalone.
It's just that implementation is common for both integration and integration integrity tests, but the purpose is a little bit different.